File: python/zoos.py
# Zoos
# https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/is-zoo-f6f309e7/

'''
# Sample code to perform I/O:

name = input()                  # Reading input from STDIN
print('Hi, %s.' % name)         # Writing output to STDOUT

# Warning: Printing unwanted or ill-formatted data to output will cause the test cases to fail
'''

# Write your code here
def main(x):
    list_x = list(x)
    z_count = list_x.count('z' or 'Z')
    o_count = list_x.count('o' or 'O')
    
    if z_count * 2 == o_count:
        print("Yes")
    else:
        print("No")


stdin_str = input()
main(stdin_str)

File: javascript/zoos.js
// Zoos
// https://www.hackerearth.com/practice/basic-programming/input-output/basics-of-input-output/practice-problems/algorithm/is-zoo-f6f309e7/

/*
// Sample code to perform I/O:

process.stdin.resume();
process.stdin.setEncoding("utf-8");
var stdin_input = "";

process.stdin.on("data", function (input) {
    stdin_input += input;                               // Reading input from STDIN
});

process.stdin.on("end", function () {
   main(stdin_input);
});

function main(input) {
    process.stdout.write("Hi, " + input + ".\n");       // Writing output to STDOUT
}

// Warning: Printing unwanted or ill-formatted data to output will cause the test cases to fail
*/

// Write your code here
process.stdin.resume();
process.stdin.setEncoding("utf-8");
var stdin_input = "";

process.stdin.on("data", function (input) {
    stdin_input += input;
});

process.stdin.on("end", function () {
   main(stdin_input);
});

function main(input) {
    var listX = input.split('');
    var zCount = 0;
    var oCount = 0;

    for (var i = 0; i < listX.length; i++) {
        if (listX[i] == 'z') {
            zCount++;
        } else if (listX[i] == 'o') {
            oCount++;
        }
    }

    if (zCount * 2 == oCount) {
        console.log("Yes");
    } else {
        console.log("No");
    }
}